src/server/check-answers/index.test.js

```diff
@@ -375,6 +340,69 @@ describe('#CheckAnswers', () => {
     expect(statusCode).toBe(statusCodes.redirect)
     expect(content).toMatchSnapshot('email-content-biosec-enabled')
   })
+
+  it('should compress the file and send email in correct format for flag enabled', async () => {
```

Comment:
bit of a nit-pick, but it should *handle* the file. It 's only getting compressed if it's over 2MB